src/EPR.Calculator.Frontend/Controllers/DashboardController.cs

```diff
@@ -27,35 +27,43 @@ public DashboardController(IConfiguration configuration, IHttpClientFactory clie
 
         public IActionResult Index()
         {
-            var dashboardcalculatorrunApi = this.configuration.GetSection("DashboardCalculatorRun").GetSection("DashboardCalculatorRunApi").Value;
-            var year = this.configuration.GetSection("DashboardCalculatorRun").GetSection("RunParameterYear").Value;
-            var client = this.clientFactory.CreateClient();
-            client.BaseAddress = new Uri(dashboardcalculatorrunApi);
-            var request = new HttpRequestMessage(HttpMethod.Post, new Uri(dashboardcalculatorrunApi));
-
-            var runParms = new CalculatorRunParamsDto
+            try
             {
-                FinancialYear = year,
-            };
-            var content = new StringContent(JsonConvert.SerializeObject(runParms), System.Text.Encoding.UTF8, "application/json");
-            request.Content = content;
-            var response = client.SendAsync(request);
-            response.Wait();
+                var dashboardcalculatorrunApi = this.configuration.GetSection("DashboardCalculatorRun").GetSection("DashboardCalculatorRunApi").Value;
+                var year = this.configuration.GetSection("DashboardCalculatorRun").GetSection("RunParameterYear").Value;
+                var client = this.clientFactory.CreateClient();
+                client.BaseAddress = new Uri(dashboardcalculatorrunApi);
+                var request = new HttpRequestMessage(HttpMethod.Post, new Uri(dashboardcalculatorrunApi));
 
-            if (response.Result.IsSuccessStatusCode)
-            {
-                var dashboardRunData = this.GetCalulationRunsData(JsonConvert.DeserializeObject<List<CalculationRun>>(response.Result.Content.ReadAsStringAsync().Result));
+                var runParms = new CalculatorRunParamsDto
+                {
+                    FinancialYear = year,
+                };
+                var content = new StringContent(JsonConvert.SerializeObject(runParms), System.Text.Encoding.UTF8, "application/json");
```

Comment:
Please create a constants for application/json

Reply:
I have updated the code. Thanks